src/services/pam/subprocess.cpp

```diff
@@ -83,7 +87,11 @@ PamIpcExitCode PamSubprocess::exec(const char* configDir, const char* config, co
 	logIf(this->log) << "Starting pam session for user \"" << user << "\" with config \"" << config
 	                 << "\" in dir \"" << configDir << "\"" << std::endl;
 
+#ifdef __FreeBSD__
+	auto result = pam_start(config, user, &conv, &handle);
+#else
 	auto result = pam_start_confdir(config, user, &conv, configDir, &handle);
+#endif
```

Comment:
If pam can't be configured on bsd a note of that should be made on the configdir property https://github.com/quickshell-mirror/quickshell/blob/3918290c1bcd93ed81291844d9f1ed146672dbfc/src/services/pam/qml.hpp#L39